frontend/src/translations/ua/common.json

```diff
@@ -0,0 +1,566 @@
+        "near_blockchain": " про \nNEAR Blockchain."
+      }
+    },
+    "accounts": {
+      "account": "Обліковий запис",
+      "error": {
+        "account_fetching": "Інформація на даний момент недоступна. Будь ласка, перевірте, чи назва облікового запису є правильною, або спробуйте пізніше.",
+        "account_not_found": "Обліковий запис ${account_id} не знайдено. Будь ласка, перевірте, чи ім'я облікового запису є правильним, або спробуйте пізніше."
+      }
+    },
+    "blocks": {
+      "title": "Блок",
+      "error": {
+        "block_fetching": "Інформація на даний момент недоступна. Будь ласка, перевірте, чи хеш блок є коректним, або спробуйте пізніше."
+      }
+    },
+    "transactions": {
+      "transaction_execution_plan": "План виконання транзакцій",
+      "error": {
+        "transaction_fetching": "Інформація на даний момент недоступна. Будь ласка, перевірте, чи хеш блок є коректним, або спробуйте пізніше."
+      }
+    }
+  },
+  "component": {
+    "nodes": {
+      "Validators": {
+        "location": "Розташування",
+        "validator": "Валідатор",
+        "fee": "Комісія",
+        "delegators": "Делегатори",
+        "stake": "Стейк",
+        "cumulative_stake": "Сукупний стейк"
+      },
+      "ValidatorRow": {
+        "warning_tip": "Валідатори 1 - ${node_tip_max} мають кумулятивну частку стейкінгу понад 33%. Делегування валідаторам нижче покращує децентралізацію мережі."
+      },
+      "ValidatorMainRow": {
+        "state": {
+          "active": {
+            "text": "поточні ноди валідаторів",
+            "title": "Активні"
+          },
+          "joining": {
+            "title": "Приєднання",
+            "text": "ноди валідаторів, які будуть в наступній епосі"
+          },
+          "leaving": {
+            "title": "Видалення",
+            "text": "ноди валідаторів, які будуть видалені в наступній епосі"
+          },
+          "proposal": {
+            "title": "Пропозиція",
+            "text": "застейкані ноди, які будуть новими нодами валідаторів"
+          },
+          "idle": {
+            "title": "Неактивні",
+            "text": "пул стейкінгу, який має менше 20% поточної ціни місця валідатора"
+          },
+          "newcomer": {
+            "title": "Новачки",
+            "text": "пул стейкінгу, який має мінімум 20% поточної ціни місця валідатора"
+          },
+          "onHold": {
+            "title": "На утриманні",
+            "text": "пул стейкінгу має достатньо токенів для валідування, але ще не подав пропозиції щодо стейкінгу"
+          }
+        }
+      },
+      "ValidatorTelemetryRow": {
+        "uptime": {
+          "text": "Час безперервної роботи оцінюється відношенням кількості створених блоків до кількості очікуваних блоків.",
+          "title": "Час безперервної роботи"
+        },
+        "latest_produced_block": {
+          "text": "Висота блоку, про яку повідомляє нод перевірки в останньому пульсі телеметрії.",
```

Comment:
Sounds very strange

Reply:
It's hard to understand the meaning in English. Any suggestions?